### DISCUSSION TITLE
「已解决」Error "Waiting for a runner to pick up this job" using GitHub Actions

### DISCUSSION THREAD

**Discussion by giscus** [youngjunning/blog.zisheng.pro#106] [Announcements]
# 「已解决」Error "Waiting for a runner to pick up this job" using GitHub Actions

当我使用 GitHub Actions 时，有一个漫长的等待，它显示 "Waiting for a runner to pick up this job"，但是这个配置已经成功运行了很多次，我不知道为什么会出现这个问题。

https://youngjuning.js.org/0c97438eff72/

**Comment by matteo-gz** [comment]
他有用对于我的工作,谢谢你的文章

**Reply by youngjunning** [reply]
很高兴能帮到你
